static/src/javascripts/projects/common/views/commercial/creatives/fabric.html

```diff
@@ -0,0 +1,17 @@
+<div
+    class="fabric"
+    style="
+        background-image: url(<%=backgroundImage%>);
+        background-color: <%=backgroundColour%>;
+        background-repeat: <%=backgroundRepeat%>;
+        background-position: <%=backgroundPosition%>;
+        background-attachment: <%=(backgroundScroll === 'reveal' ? 'fixed' : 'scroll')%>;
```

Comment:
TBH I'm not sure but I'd say the quotes are dangerous. They are probably escaped in a way that does not bode well with `template`. It will convert this into a string that is then fed into a `Function` constructor, meta-programming ftw.